packages/rsx/src/ifmt.rs

```diff
@@ -399,10 +391,11 @@ mod tests {
         let input = syn::parse2::<IfmtInput>(quote! { "hello {world} {world} {world()}" }).unwrap();
         println!("{}", input.to_string_with_quotes());
 
+        // raw expressions are always static!
         let input =
             syn::parse2::<IfmtInput>(quote! { r#"hello {world} {world} {world()}"# }).unwrap();
         println!("{}", input.to_string_with_quotes());
-        assert!(!input.is_static());
+        assert!(input.is_static());
```

Comment:
This seems wrong... The input has a bunch of dynamic segments. It shouldn't be static